lib/seam/resources/warning.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Seam
+  class Warning < BaseResource
```

Comment:
Otherwise it's not consistent with the `Error`.  (See last comment) Need to update refs as well.

```suggestion
  class ResourceWarning < BaseResource
```